builder/builder-next/worker/worker.go

```diff
@@ -50,7 +50,7 @@ import (
 )
 
 func init() {
-	version.Version = "v0.11.6"
+	version.Version = "v0.11.7-0.20230519102302-348e79dfed17"
```

Comment:
Do we prefer the go module version for this, or `git describe` ?

```bash
git describe --tags
v0.11.6-4-g348e79dfe
```

Reply:
Not near my PC but maybe we could backport https://github.com/moby/moby/pull/45304 to 24.0 branch, WDYT?

Reply:
Yeah, didn't want to jinx it. And it looks like we don't need this intermediate version, so updating it to v0.11.7 (once it's released) isn't _too much_ of a hassle for now I guess.